oscola.csl

```diff
@@ -202,8 +202,8 @@
       <if type="book legislation motion_picture manuscript" match="any">
         <text variable="title" font-style="italic" text-case="title" form="short"/>
```

Comment:
@fbennett : The title-short for legislations will then be still title case (and italic). Is that okay?